src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+
+{{ figure_markup(
+  image="video-durations.png",
+  caption="Video durations.",
+  description="A bar chart showing how long videos are, on desktop and mobile. On desktop, 21% of videos are between 0-10 seconds, 39% are between 10-30 seconds, 19% are between 30-60 seconds, 12% are between 60-120 seconds, and 10% are over 120 seconds. On mobile, 21% are between 0-10 seconds, 36% are between 10-30 seconds, 20% are between 30-60 seconds, 13% are between 60-120 seconds, and 11% are more than 120 seconds.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=366287434&format=interactive",
+  sheets_gid="1864781514",
+  sql_file="video_durations_mobile.sql"
+  )
+}}
+
+Most videos on the web are short: ~ 60% of videos are under 30 seconds long on both mobile and desktop. However, 12-13% are between one and two minutes, and 10% of videos are over two minutes long.
+
+### Video: formats
+
+What types of files are being delivered as video?  We queried all files with "video" in the MIME type, and then sorted by the file extension.
+
+The chart below shows all video extensions with over 1% market share:
+
+{{ figure_markup(
+  image="top-extensions-of-files-with-a-video-mime-type.png",
+  caption="Top extensions of files with a video MIME type.",
+  description="A bar chart showing the extensions of video files delivered, on desktop and mobile, in both 2020 and 2021. Only the desktop numebrs are labeled; the mobile numbers are similar. The mp4 extension accounted for 64.31% of videos on desktop in 2020, but only 49.22% in 2021. Blank extensions accounted for 17.76% of videos in 2020, increasing sharply to 32.12% in 2021. The ts extension accounted for 6.27% of fvideos in 2020, and 10.04% in 2021. m4s rose from 2ish% (the label is obscured) to 3.34%. webm fell from 7.62% to 3.28%. Finally, mov fell from 0.83% in 2020 to an unlabelled, almost invisible bar that looks to be approximately 0% in 2021.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=2066681624&format=interactive",
+  sheets_gid="1430902287",
+  sql_file="video_ext.sql"
+  )
+}}
+
+By far, the #1 video format on the web is the mp4, since the <a hreflang="en" href="https://caniuse.com/mpeg4">mp4 h264 format has 98.4% support in all modern browsers</a>, and the 1.9% of browsers that do not support mp4 have no video support, so the number is really 100% coverage. Interestingly, the mp4 usage has dropped by ~15% YOY on both desktop and mobile. WebM support also dropped significantly YOY (50% drop on both mobile and desktop). (data from [Web Almanac 2020](../2020/media#videos)
```

Comment:
```suggestion
By far, the #1 video format on the web is the `mp4` (or MPEG-4), since the <a hreflang="en" href="https://caniuse.com/mpeg4">mp4 h264 format has 98.4% support in all modern browsers</a>, and the 1.9% of browsers that do not support `mp4` have no video support, so the number is really 100% coverage. Interestingly, the mp4 usage has dropped by ~15% YOY on both desktop and mobile. WebM support also [dropped significantly YOY](../2020/media#videos) (50% drop on both mobile and desktop).
```

Reply:
Applied these edits.